coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -406,7 +406,7 @@ def _convolution(context, node):
 
     dilations = inputs[5]
     out_pad = None
-    if len(inputs) == 12:
+    if len(inputs) >= 12:
```

Comment:
Hi @skyline75489, A different version of PyTorch might have a different number of inputs, do you know what's extra in your case? Looks like it's not impact anything if we ignore, but we still need to note things down and make sure we're not ignoring important inputs and result in an error in different versions.

Reply:
We should let gitesh and @aseemw  weigh in on this. PyTorch adds parameters in the end, so existing parameters are okay. 

The question is we should ignore new parameters by default or whether we should manually update code to ignore them. I feel that it's more common to have new params for optimization / corner cases that doesn't impact the existing implementation, so relaxing this is probably okay. (I know aseem and gitesh has thought about doing this)

Reply:
Originally I was using the 12 or 13 check. Then I thought why not relax the restriction in case in the future more other parameters are added. So yeah here it is. I’m personally ok either way, as long as it doesn’t crash.

Reply:
Yes , in my opinion , the check `>=12` should be fine. 
@skykine75489, can you please rebase this change on top of master, since #1020 was recently merged, which touched this piece of code as well.